app/src/main/java/movierating/services/AverageReviewScoreForMovie.java

```diff
@@ -0,0 +1,26 @@
+package movierating.services;
+
+import movierating.models.Movie;
+import movierating.repositories.MovieRepository;
+
+public class AverageReviewScoreForMovie {
+
+    private final MovieRepository movieRepository;
+
+    public AverageReviewScoreForMovie(MovieRepository movieRepository) {
+        this.movieRepository = movieRepository;
+    }
+
+    public int getAverageReviewScoreByYear(int releaseYear){
```

Comment:
Why is average a int? Shouldn't it be 3.5?

Reply:
Ya. Fixed this.